edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/logs/LogMessageParser.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core.Logs
+{
+    extern alias akka;
+    using System;
+    using System.Text;
+    using System.Text.RegularExpressions;
+    using akka::Akka.IO;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    // Parses logs into message objects.
+    //
+    // Expected format -
+    // Each input payload should contain one frame in Docker format -
+    //    01 00 00 00 00 00 00 1f 52 6f 73 65 73 20 61 72  65 ...
+    //    │  ─────┬── ─────┬─────  R o  s e  s a  r e...
+    //    │       │        │
+    //    └stdout │        │
+    //            │        └─ 0x0000001f = 31 bytes (including the \n at the end)
+    //         unused
+    //
+    // The payload itself is expected to be in this format -
+    // <logLevel> [TimeStamp] log text
+    // For example, this log line will be parsed as follows -
+    // <6> 2019-02-14 16:15:35.243 -08:00 [INF] [EdgeHub] - Version - 1.0.7-dev.BUILDNUMBER (COMMITID)
```

Comment:
what is the difference between "<6>" and "[INF]"? does it represent same information?

Reply:
Yes, it is duplicated. But the <6> is for parsing by tools etc.

Reply:
why we print <6>? does this show in current edge agent/hub log?

Reply:
Yes.. it is for tools that would like to parse it.

Reply:
It is the standard syslog format, and as per Mike's suggestion. I am sending out a PR for this (after my other PR is merged)